grizzly/common/status_reporter.py

```diff
@@ -242,27 +247,28 @@ def summary(
             total_iters = sum(iterations)
 
             # Iterations
-            disp = []
-            disp.append(str(total_iters))
+            disp = [str(total_iters)]
             if count > 1:
                 disp.append(f" ({max(iterations)}, {min(iterations)})")
             entries.append(("Iterations", "".join(disp)))
 
             # Rate
-            disp = []
-            disp.append(f"{count} @ {round(sum(rates), 2):0.2f}")
-            if count > 1:
-                disp.append(
-                    f" ({round(max(rates), 2):0.2f}, {round(min(rates), 2):0.2f})"
-                )
-            entries.append(("Rate", "".join(disp)))
+            if rate:
+                disp = [f"{count} @ {round(sum(rates), 2):0.2f}"]
```

Comment:
I realize this isn't changed code, but this pattern is unnecessary (you don't need `round()`):

```
>>> f"{round(0.5555,2)}"
'0.56'
>>> f"{round(0.5555,2):0.2f}"
'0.56'
>>> f"{0.5555:0.2f}"
'0.56'
>>>
```

Reply:
Good catch, thanks.